build.sbt

```diff
@@ -7,6 +7,18 @@ lazy val backend = (project in file("backend"))
   .dependsOn(core % "test->test;compile->compile")
   .settings(backendSettings:_*)
 
+lazy val localBackend = (project in file("local-backend"))
```

Comment:
I couldn't see any reason not to. They'll be aggregated in the global cromwell jar anyway, but they'll also have their own jar which will give us the ability to fix/update only one backend if necessary. Plus it helps validating that this one jar per backend model will work with future external backends